awx_collection/plugins/modules/tower_job_launch.py

```diff
@@ -114,11 +139,11 @@ def main():
         credential=dict(type='str', default=None),
         limit=dict(),
         tags=dict(type='list'),
-        extra_vars=dict(type='list'),
+        extra_vars=dict(type='dict', required=False),
```

Comment:
I think we should add an entry in

https://github.com/ansible/awx/blob/414cebd9429f0130e6806ac9229e56530ab2127a/awx_collection/README.md

under "Release and Upgrade Notes", saying that maybe this used to work with a list (we're not really sure), but now it takes a dict and only a dict.